main.html

```diff
@@ -12,21 +12,28 @@
     />
     <link rel="stylesheet" href="style.css" />
 
-    <title>Awesome Books by Rudi & Kevin</title>
+    <title>All Awesome Books by Rudi & Kevin</title>
   </head>
     <body>
+      <header>
+        <nav class="navbar">
+          <a class="logo" href="main.html">Awesome books</a>
+          <ul class="lists">
+            <li><a href="main.html">List</a></li>
+            <li><a href="addbook.html">Add new</a></li>
+            <li><a href="contact.html">Contact</a></li>
+          </ul>
```

Comment:
- [OPTIONAL] Good job working on some of the project requirements but I noticed your design is missing a few things that need to be addressed before your work gets approval. If you carefully look at the template design or wireframe it displays the date and time in all page tabs but for your design, it is not displaying the date and time. This was supposed to be done using [luxon](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)

This is how your work looks:



![image](https://github.com/leloufadel/AwesomeBook/assets/79058364/90718345-43f6-4508-ab30-3109013b6281)

Please follow this wireframe to display the date and time:

![DATE AND TIME ](https://github.com/kamranasif506/Awesome-books/assets/79506105/506a66fe-e959-412a-b33b-def74caa00d1)

Remember this should appear on all three pages of your tabs.


- [ ] Secondly it would be nice to properly align the content on your contact page to properly match the design template you can do this by adjusting the margin and padding.

your page content should be centered properly. Please follow this template design:

![Contact page ](https://github.com/kamranasif506/Awesome-books/assets/79506105/528e841b-9bf1-4aaf-8b04-bc5c8861714f)

Reply:
@BenMukebo hi! the luxon add you're asking us for is not on the requirements of the project, also we haven't seen anything about luxon yet, and on the requirements, they ask us to use the template as a GUIDE not to replicate it...